de.dlr.sc.virsat.commons/src/de/dlr/sc/virsat/commons/command/InputStreamConsumer.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.commons.command;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class InputStreamConsumer extends Thread {
```

Comment:
What is this class about? reading from a stream and writing back to a buffered writer?